### ISSUE TITLE
Add kn client cheatsheet

### ISSUE THREAD

**Issue by kameshsampath** [redhat-developer/cheat-sheets#7]
Add a new cheatsheet show use of Knative Client (`kn`).

**Comment by jonhorton** [comment]
Hi @kameshsampath! We'd love to have you add a cheatsheet to show use of Knative Client so that we can publish and promote to our developer audience. Is this something you'd be able to contribute?

**Comment by kameshsampath** [comment]
yes I should be doing soon .. just waiting on some more stable releases of it
